### PR TITLE
Os 625 fixed accessibility issues at schedules page

### PR DESCRIPTION
**Pull request by retif**
**Steps to Reproduce:**
When using a keyboard, tab to the "add to calendar" link and press enter. Focus is placed on a list of calendar options, however, when I tested there were five lists of calendar options to tab through. Examining the code also showed 5 lists of calendar options i.e. iCal, Google, Yahoo, etc.

**Actual Results**
Focus is correctly placed on the list of calendar options, but tab and shift-tab to find there are five lists of options in a row.Expected REsultsThe opening "add to calendar" should only list the calendar options one time so as to reduce tab stops and to eliminate confusion.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

File: themes/openy_themes/openy_lily/css/style.css.map [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...,IAAI;EACf,MAAM,EAAE,IAAI;EACZ,WAAW,EAAE,IAAI;EACjB,cAAc,EAAE,SAAS;EACzB,WAAW,EAAE,MAAM;EACnB,OAAO,EAAE,MAAM;EACf,MAAM,EAAE,YAAY;;A1BhKxB,wBAAwB;E0BoJtB,gEAAQ;IAcJ,MAAM,EAAE,cAAc;IACtB,KAAK,E9BxNH,OAAO;;;A8B0NX,wEAAQ;EACN,WAAW,EAAE,IAAI;EACjB,WAAW,EAAE,IAAI;;A1BvKvB,wBAAwB;E0BqKpB,wEAAQ;IAIJ,WAAW,EAAE,IAAI;;;AAEnB,4EAAI;EACF,SAAS,EAAE,IAAI;EACf,KAAK,EnB9FV,OAAO;;AP/EV,wBAAwB;E0B2KlB,4EAAI;IAIA,SAAS,EAAE,IAAI;;;AASzB,8GAAyB;EACvB,KAAK,E9B9OC,OAAO;E8B+Ob,UAAU,E9BnPD,OAAO;;;A8ByPpB,0CAAS;EACP,aAAa,EAAE,IAAI;;AACnB,6CAAG;EACD,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,aAAa,EAAE,IAAI;EACnB,WAAW,EAAE,GAAG;;A1BtMlB,wBAAwB;E0BkMxB,6CAAG;IAOC,SAAS,EAAE,IAAI;IACf,WAAW,EAAE,IAAI;;;AAEnB,gDAAG;EACD,UAAU,EAAE,MAAM;;AAItB,sDAAY;E7B5Qd,WAAW,EAAE,qCAAqC;E6B8Q9C,KAAK,EnBpMF,OAAO;EmBqMV,SAAS,EAAE,IAAI;EACf,aAAa,EAAE,IAAI;EACnB,UAAU,EAAE,CAAC;EACb,WAAW,EAAE,OAAO;EACpB,WAAW,EAAE,MAAM;;A1BxNrB,wBAAwB;E0BiNxB,sDAAY;IASR,SAAS,EAAE,IAAI;;;AAGnB,0EAAgC;EAC9B,MAAM,EAAE,QAAQ;;A1B9NlB,wBAAwB;E0B6NxB,0EAAgC;IAG5B,MAAM,EAAE,WAAW;;;A1BhOvB,2BAAwB;E0B6NxB,0EAAgC;IAM5B,YAAY,EAAE,CAAC;;;AAEjB,uFAAa;EACX,OAAO,EAAE,WAAW;EACpB,OAAO,EAAE,YAAY;EACrB,OAAO,EAAE,WAAW;EACpB,OAAO,EAAE,IAAI;EACb,SAAS,EAAE,IAAI;;AAEjB,uFAAa;EACX
```
Comment (podarok) [inline themes/openy_themes/openy_lily/css/style.css.map]: map is a dev feature, let's remove it from production, please

**Review by podarok — CHANGES_REQUESTED** [review]
Small thing to get it merged

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2011/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2011/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2011/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2011/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2011/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2011/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2011/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2011/

**Comment by gundevel** [comment]
!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2017/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1885 of commit 07f4a597960c0236b05ed05f64db0ff862269bfe automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2017/openy-profile # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git2855414292822563939.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
ERROR: Error cloning remote repo 'origin'
hudson.plugins.git.GitException: Command "git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1" returned status code 128:
stdout: 
stderr: remote: Enumerating objects: 14109, done.
remote: Counting objects:   0% (1/14109)
remote: Counting objects:   1% (142/14109)
remote: Counting objects:   2% (283/14109)
remote: Counting objects:   3% (424/14109)
remote: Counting objects:   4% (565/14109)
remote: Counting objects:   5% (706/14109)
remote: Counting objects:   6% (847/14109)
remote: Counting objects:   7% (988/14109)
remote: Counting objects:   8% (1129/14109)
remote: Counting objects:   9% (1270/14109)
remote: Counting objects:  10% (1411/14109)
remote: Counting objects:  11% (1552/14109)
remote: Counting objects:  12% (1694/14109)
remote: Counting objects:  13% (1835/14109)
remote: Counting objects:  14% (1976/14109)
remote: Counting objects:  15% (2117/14109)
remote: Counting objects:  16% (2258/14109)
remote: Counting objects:  17% (2399/14109)
remote: Counting objects:  18% (2540/14109)
remote: Counting objects:  19% (2681/14109)
remote: Counting objects:  20% (2822/14109)
remote: Counting objects:  21% (2963/14109)
remote: Counting objects:  22% (3104/14109)
remote: Counting objects:  23% (3246/14109)
remote: Counting objects:  24% (3387/14109)
remote: Counting objects:  25% (3528/14109)
remote: Counting objects:  26% (3669/14109)
remote: Counting objects:  27% (3810/14109)
remote: Counting objects:  28% (3951/14109)
remote: Counting objects:  29% (4092/14109)
remote: Counting objects:  30% (4233/14109)
remote: Counting objects:  31% (4374/14109)
remote: Counting objects:  32% (4515/14109)
remote: Counting objects:  33% (4656/14109)
remote: Counting objects:  34% (4798/14109)
remote: Counting objects:  35% (4939/14109)
remote: Counting objects:  36% (5080/14109)
remote: Counting objects:  37% (5221/14109)
remote: Counting objects:  38% (5362/14109)
remote: Counting objects:  39% (5503/14109)
remote: Counting objects:  40% (5644/14109)
remote: Counting objects:  41% (5785/14109)
remote: Counting objects:  42% (5926/14109)
remote: Counting objects:  43% (6067/14109)
remote: Counting objects:  44% (6208/14109)
remote: Counting objects:  45% (6350/14109)
remote: Counting objects:  46% (6491/14109)
remote: Counting objects:  47% (6632/14109)
remote: Counting objects:  48% (6773/14109)
remote: Counting objects:  49% (6914/14109)
remote: Counting objects:  50% (7055/14109)
remote: Counting objects:  51% (7196/14109)
remote: Counting objects:  52% (7337/14109)
remote: Counting objects:  53% (7478/14109)
remote: Counting objects:  54% (7619/14109)
remote: Counting objects:  55% (7760/14109)
remote: Counting objects:  56% (7902/14109)
remote: Counting objects:  57% (8043/14109)
remote: Counting objects:  58% (8184/14109)
remote: Counting objects:  59% (8325/14109)
remote: Counting objects:  60% (8466/14109)
remote: Counting objects:  61% (8607/14109)
remote: Counting objects:  62% (8748/14109)
remote: Counting objects:  63% (8889/14109)
remote: Counting objects:  64% (9030/14109)
remote: Counting objects:  65% (9171/14109)
remote: Counting objects:  66% (9312/14109)
remote: Counting objects:  67% (9454/14109)
remote: Counting objects:  68% (9595/14109)
remote: Counting objects:  69% (9736/14109)
remote: Counting objects:  70% (9877/14109)
remote: Counting objects:  71% (10018/14109)
remote: Counting objects:  72% (10159/14109)
remote: Counting objects:  73% (10300/14109)
remote: Counting objects:  74% (10441/14109)
remote: Counting objects:  75% (10582/14109)
remote: Counting objects:  76% (10723/14109)
remote: Counting objects:  77% (10864/14109)
remote: Counting objects:  78% (11006/14109)
remote: Counting objects:  79% (11147/14109)
remote: Counting objects:  80% (11288/14109)
remote: Counting objects:  81% (11429/14109)
remote: Counting objects:  82% (11570/14109)
remote: Counting objects:  83% (11711/14109)
remote: Counting objects:  84% (11852/14109)
remote: Counting objects:  85% (11993/14109)
remote: Counting objects:  86% (12134/14109)
remote: Counting objects:  87% (12275/14109)
remote: Counting objects:  88% (12416/14109)
remote: Counting objects:  89% (12558/14109)
remote: Counting objects:  90% (12699/14109)
remote: Counting objects:  91% (12840/14109)
remote: Counting objects:  92% (12981/14109)
remote: Counting objects:  93% (13122/14109)
remote: Counting objects:  94% (13263/14109)
remote: Counting objects:  95% (13404/14109)
remote: Counting objects:  96% (13545/14109)
remote: Counting objects:  97% (13686/14109)
remote: Counting objects:  98% (13827/14109)
remote: Counting objects:  99% (13968/14109)
remote: Counting objects: 100% (14109/14109)
remote: Counting objects: 100% (14109/14109), done.
remote: Compressing objects:   0% (1/7859)
remote: Compressing objects:   1% (79/7859)
remote: Compressing objects:   2% (158/7859)
remote: Compressing objects:   3% (236/7859)
remote: Compressing objects:   4% (315/7859)
remote: Compressing objects:   5% (393/7859)
remote: Compressing objects:   6% (472/7859)
remote: Compressing objects:   7% (551/7859)
remote: Compressing objects:   8% (629/7859)
remote: Compressing objects:   9% (708/7859)
remote: Compressing objects:  10% (786/7859)
remote: Compressing objects:  11% (865/7859)
remote: Compressing objects:  12% (944/7859)
remote: Compressing objects:  13% (1022/7859)
remote: Compressing objects:  14% (1101/7859)
remote: Compressing objects:  15% (1179/7859)
remote: Compressing objects:  16% (1258/7859)
remote: Compressing objects:  17% (1337/7859)
remote: Compressing objects:  18% (1415/7859)
remote: Compressing objects:  19% (1494/7859)
remote: Compressing objects:  20% (1572/7859)
remote: Compressing objects:  21% (1651/7859)
remote: Compressing objects:  22% (1729/7859)
remote: Compressing objects:  23% (1808/7859)
remote: Compressing objects:  24% (1887/7859)
remote: Compressing objects:  25% (1965/7859)
remote: Compressing objects:  26% (2044/7859)
remote: Compressing objects:  27% (2122/7859)
remote: Compressing objects:  28% (2201/7859)
remote: Compressing objects:  29% (2280/7859)
remote: Compressing objects:  30% (2358/7859)
remote: Compressing objects:  31% (2437/7859)
remote: Compressing objects:  32% (2515/7859)
remote: Compressing objects:  33% (2594/7859)
remote: Compressing objects:  34% (2673/7859)
remote: Compressing objects:  35% (2751/7859)
remote: Compressing objects:  36% (2830/7859)
remote: Compressing objects:  37% (2908/7859)
remote: Compressing objects:  38% (2987/7859)
remote: Compressing objects:  39% (3066/7859)
remote: Compressing objects:  40% (3144/7859)
remote: Compressing objects:  41% (3223/7859)
remote: Compressing objects:  42% (3301/7859)
remote: Compressing objects:  43% (3380/7859)
remote: Compressing objects:  44% (3458/7859)
remote: Compressing objects:  45% (3537/7859)
remote: Compressing objects:  46% (3616/7859)
remote: Compressing objects:  47% (3694/7859)
remote: Compressing objects:  48% (3773/7859)
remote: Compressing objects:  49% (3851/7859)
remote: Compressing objects:  50% (3930/7859)
remote: Compressing objects:  51% (4009/7859)
remote: Compressing objects:  52% (4087/7859)
remote: Compressing objects:  53% (4166/7859)
remote: Compressing objects:  54% (4244/7859)
remote: Compressing objects:  55% (4323/7859)
remote: Compressing objects:  56% (4402/7859)
remote: Compressing objects:  57% (4480/7859)
remote: Compressing objects:  58% (4559/7859)
remote: Compressing objects:  59% (4637/7859)
remote: Compressing objects:  60% (4716/7859)
remote: Compressing objects:  61% (4794/7859)
remote: Compressing objects:  62% (4873/7859)
remote: Compressing objects:  63% (4952/7859)
remote: Compressing objects:  64% (5030/7859)
remote: Compressing objects:  65% (5109/7859)
remote: Compressing objects:  66% (5187/7859)
remote: Compressing objects:  67% (5266/7859)
remote: Compressing objects:  68% (5345/7859)
remote: Compressing objects:  69% (5423/7859)
remote: Compressing objects:  70% (5502/7859)
remote: Compressing objects:  71% (5580/7859)
remote: Compressing objects:  72% (5659/7859)
remote: Compressing objects:  73% (5738/7859)
remote: Compressing objects:  74% (5816/7859)
remote: Compressing objects:  75% (5895/7859)
remote: Compressing objects:  76% (5973/7859)
remote: Compressing objects:  77% (6052/7859)
remote: Compressing objects:  78% (6131/7859)
remote: Compressing objects:  79% (6209/7859)
remote: Compressing objects:  80% (6288/7859)
remote: Compressing objects:  81% (6366/7859)
remote: Compressing objects:  82% (6445/7859)
remote: Compressing objects:  83% (6523/7859)
remote: Compressing objects:  84% (6602/7859)
remote: Compressing objects:  85% (6681/7859)
remote: Compressing objects:  86% (6759/7859)
remote: Compressing objects:  87% (6838/7859)
remote: Compressing objects:  88% (6916/7859)
remote: Compressing objects:  89% (6995/7859)
remote: Compressing objects:  90% (7074/7859)
remote: Compressing objects:  91% (7152/7859)
remote: Compressing objects:  92% (7231/7859)
remote: Compressing objects:  93% (7309/7859)
remote: Compressing objects:  94% (7388/7859)
remote: Compressing objects:  95% (7467/7859)
remote: Compressing objects:  96% (7545/7859)
remote: Compressing objects:  97% (7624/7859)
remote: Compressing objects:  98% (7702/7859)
remote: Compressing objects:  99% (7781/7859)
remote: Compressing objects: 100% (7859/7859)
remote: Compressing objects: 100% (7859/7859), done.
Receiving objects:   0% (1/14109)   
Receiving objects:   1% (142/14109)   
Receiving objects:   2% (283/14109)   
Receiving objects:   3% (424/14109)   
Receiving objects:   4% (565/14109)   
Receiving objects:   5% (706/14109)   
Receiving objects:   6% (847/14109)   
Receiving objects:   7% (988/14109)   
Receiving objects:   8% (1129/14109)   
Receiving objects:   9% (1270/14109)   
Receiving objects:  10% (1411/14109)   
Receiving objects:  11% (1552/14109)   
Receiving objects:  12% (1694/14109)   
Receiving objects:  13% (1835/14109)   
Receiving objects:  14% (1976/14109)   
Receiving objects:  15% (2117/14109)   
Receiving objects:  16% (2258/14109)   
Receiving objects:  17% (2399/14109)   
Receiving objects:  18% (2540/14109)   
Receiving objects:  19% (2681/14109)   
Receiving objects:  20% (2822/14109)   
Receiving objects:  21% (2963/14109)   
Receiving objects:  22% (3104/14109)   
Receiving objects:  23% (3246/14109)   
Receiving objects:  24% (3387/14109)   
Receiving objects:  25% (3528/14109)   
Receiving objects:  26% (3669/14109)   
Receiving objects:  27% (3810/14109)   
Receiving objects:  28% (3951/14109)   
Receiving objects:  29% (4092/14109)   
Receiving objects:  30% (4233/14109)   
Receiving objects:  31% (4374/14109)   
Receiving objects:  32% (4515/14109)   
Receiving objects:  33% (4656/14109)   
Receiving objects:  34% (4798/14109)   
Receiving objects:  35% (4939/14109)   
Receiving objects:  36% (5080/14109)   
Receiving objects:  37% (5221/14109)   
Receiving objects:  38% (5362/14109)   
Receiving objects:  39% (5503/14109)   
Receiving objects:  40% (5644/14109)   
Receiving objects:  41% (5785/14109)   
Receiving objects:  42% (5926/14109)   
Receiving objects:  43% (6067/14109)   
Receiving objects:  44% (6208/14109)   
Receiving objects:  45% (6350/14109)   
Receiving objects:  46% (6491/14109)   
Receiving objects:  47% (6632/14109)   
Receiving objects:  48% (6773/14109)   
Receiving objects:  49% (6914/14109)   
Receiving objects:  50% (7055/14109)   
Receiving objects:  51% (7196/14109)   
Receiving objects:  52% (7337/14109)   
fatal: write error: No space left on device
fatal: index-pack failed

	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:1600)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandWithCredentials(CliGitAPIImpl.java:1363)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.access$300(CliGitAPIImpl.java:61)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$1.execute(CliGitAPIImpl.java:299)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$2.execute(CliGitAPIImpl.java:485)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1003)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1043)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1277)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:610)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:532)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: null
Couldn't read comment file
java.io.FileNotFoundException: File '/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2017/commentinfo.md' does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1748)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:124)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:201)
	at hudson.model.Run.execute(Run.java:1786)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
```

**Comment by gundevel** [comment]
!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2019/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1885 of commit f899cec584c86cb8e31f6bf54562a8898035f717 automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2019/openy-profile # timeout=10
ERROR: Error cloning remote repo 'origin'
hudson.plugins.git.GitException: Could not init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2019/openy-profile
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$4.execute(CliGitAPIImpl.java:585)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$2.execute(CliGitAPIImpl.java:442)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1003)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1043)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1277)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:610)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:532)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
Caused by: hudson.plugins.git.GitException: Command "git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2019/openy-profile" returned status code 128:
stdout: 
stderr: error: copy-fd: write returned No space left on device
fatal: cannot copy '/usr/share/git-core/templates/hooks/pre-push.sample' to '/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2019/openy-profile/.git/hooks/pre-push.sample': No space left on device

	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:1600)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:1576)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:1572)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommand(CliGitAPIImpl.java:1233)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$4.execute(CliGitAPIImpl.java:583)
	... 12 more
ERROR: null
Couldn't read comment file
java.io.FileNotFoundException: File '/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2019/commentinfo.md' does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1748)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:124)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:201)
	at hudson.model.Run.execute(Run.java:1786)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
```

**Comment by podarok** [comment]
@retif please fix conflicts

**Review by anpolimus — DISMISSED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2060/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2060/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2060/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2060/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2060/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2060/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2060/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
@retif one more time conflicts fix needed, please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2071/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2071/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2071/

**Pull request merged**